source/ios/AdaptiveCards/ADCIOSVisualizer/resources/sample.json

```diff
@@ -11,7 +11,7 @@
 	"imageBaseUrl": "https://pbs.twimg.com/profile_images/3647943215/",
 	"fontTypes": {
 		"default": {
-			"fontFamily": "'Segoe UI', 'sans-serif', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue'",
+			"fontFamily": "San Francisco, American Typewriter, Segoe UI, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue",
```

Comment:
we should probably document that dynamic font size changes for hosts requires use of Apple's new fonts...

Reply:
https://developer.apple.com/documentation/uikit/uifont/scaling_fonts_automatically?language=objc
`Although custom fonts are supported in Dynamic Type, the preferred font is designed to look good at any size (for more information, see Typography). Also, using the preferred font ensures consistency within the system and with other apps.`